itest/rust/src/register_tests/constant_test.rs

```diff
@@ -28,6 +31,14 @@ impl HasConstants {
     #[constant]
     #[rustfmt::skip]
     const DONT_PANIC_WITH_SEGMENTED_PATH_ATTRIBUTE: bool = true;
+
+    #[cfg(all())]
+    #[constant]
+    const CONST_SHOULD_STILL_BE_RECOGNIZED_WITH_SIMPLE_PATH_ATTRIBUTE_ABOVE_CONST_ATTR: bool = true;
+
+    #[constant]
+    #[cfg(all())]
+    const CONST_SHOULD_STILL_BE_RECOGNIZED_WITH_SIMPLE_PATH_ATTRIBUTE_UNDER_CONST_ATTR: bool = true;
```

Comment:
I'd use "below" instead of "under", and "constant" instead of "const".

Also, can the identifiers be a bit shorter? "should still be recognized" can be shortened to just "recognized".

Same applies to functions 🙂

Reply:
Done, and done! :grin: